### PR TITLE
squid:ClassVariableVisibilityCheck - Class variable fields should not have public accessibility

### PR DESCRIPTION
**Pull request by georgekankava**
This pull request is focused on resolving occurrence of Sonar rule
squid:ClassVariableVisibilityCheck - Class variable fields should not have public accessibility.
This pull request removes 60 minutes of technical debt.
You can find more information about the issue here:
https://dev.eclipse.org/sonar/rules/show/squid:ClassVariableVisibilityCheck
Please let me know if you have any questions.
George Kankava

### CODE REVIEW

File: app/src/androidTest/java/com/zulip/android/test/mutated/FakeAsyncGetOldMessages.java [hunk 1]
```diff
@@ -69,4 +69,51 @@ protected void onPostExecute(String result) {
...
+
+    public int getFmNumAfter() {
+        return fmNumAfter;
+    }
+
+    public void setFmNumAfter(int fmNumAfter) {
+        this.fmNumAfter = fmNumAfter;
+    }
+
+    public boolean isShouldFmSucceed() {
```
Comment (niftynei) [inline app/src/androidTest/java/com/zulip/android/test/mutated/FakeAsyncGetOldMessages.java]: Gramatically, `shouldFmSucceed` is correct. Adding `is` is unnecessary here.

**Comment by niftynei** [comment]
other than the comment, looks good to me.

**Comment by georgekankava** [comment]
@niftynei Thanks for the comment. Updated.

**Comment by niftynei** [comment]
This has been merged. Thanks!

**Pull request closed without merging**